src/hooks/query/event/useDeleteEventMutation.ts

```diff
@@ -1,18 +1,26 @@
 import deleteEvent from '@/apis/event/deleteEvent';
 import useToast from '@/hooks/useToast';
 
-import { useMutation } from '@tanstack/react-query';
+import { useNavigate } from 'react-router-dom';
+
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+
+import { QUERY_KEY as GET_ALL_EVENT_QUERY_KEY } from './useAllEventsQuery';
 
 interface useDeleteEventMutation {
   eventId: string;
 }
 
 const useDeleteEventMutation = ({ eventId }: useDeleteEventMutation) => {
   const { createToast } = useToast();
+  const queryClient = useQueryClient();
+  const navigate = useNavigate();
   const { mutate: deleteEventMutate } = useMutation({
     mutationKey: ['delete_event', `${eventId}`],
     mutationFn: () => deleteEvent({ eventId }),
     onSuccess: () => {
+      queryClient.invalidateQueries({ queryKey: [GET_ALL_EVENT_QUERY_KEY] });
+      navigate('/');
```

Comment:
```suggestion
      navigate(PATH.MAIN);
```
상수화 된 PATH를 사용하면 좋을것 같아요!

Reply:
넵 반영하겠습니다!